generators/app/templates/src/containers/Profile/components/Image/image.component.js

```diff
@@ -77,7 +77,7 @@ export const Image = ({ webId, defaultProfilePhoto }: Props) => {
         },
         onError: error => {
           if (error && error.statusText) {
-            errorToaster(error.statusText);
+            errorToaster(error.statusText, 'Error');
```

Comment:
Title should be translated instead of hardcoded

Reply:
Fixed